components/shared/indexIssueDetails/IndexIssueDetails.js

```diff
@@ -19,7 +19,10 @@ const markdownComponents = {
   // This custom renderer changes how images are rendered
   // we use it to constrain the max width of an image to its container
   img: ({ node, ...props }) => (
-    <Image className="img-fluid" alt="image from github" {...props} />
+    <Image {...props} className="img-fluid" alt="image from github"/>
+  ),
+  a: ({ node, ...props }) => (
+    <a {...props} className="markdown-link"/>
```

Comment:
also, fwiw once you merge that other branch this will become a non-issue. so it's also fine to do nothing and wait :)

Reply:
@johnclary Just implemented all your suggestions! Thanks so much. I realized I should have cleaned up + double checked my code before pushing, since it went through several rounds of revision. Bad habits on my end.

Reply:
@johnclary I just implemented all your suggestions; thanks so much! I realized I should have cleaned up+double checked my code before pushing since it went through several times of revision. Will do next time. I tried `git rebase` once, but it ended up giving me some branching issues and asking me to git pull from this repo. I was afraid of losing track again, so chose the route of letting the previous commit stay there to wait and see. Please let me know if that's not ok

Reply:
that's perfect—no problem!

Reply:
@johnclary `rebase` is my enemy i avoid it at all costs :imp: